packages/webpack-loader/src/compiled-loader.ts

```diff
@@ -105,6 +109,7 @@ function getLoaderOptions(context: LoaderContext<CompiledLoaderOptions>) {
     classNameCompressionMap,
     extractStylesToDirectory,
     resolver,
+    importSources,
```

Comment:
I don't know anything about this, I presume this whole object `getLoaderOptions` gets passed into Babel?

Reply:
yep! `@compiled/webpack-loader` is acts as a kind of wrapper around `@compiled/babel-plugin` and `@compiled/babel-plugin-strip-runtime`